website/docs/main-concepts.mdx

```diff
@@ -6,7 +6,7 @@ description: This page explains the basics of feature flags and how to use them.
 
 ## Feature Flag or Setting
 
-A _Setting_ is an essential bit of _ConfigCat_. It can be of multiple types like on/off switch (bool), number (int, double) or any text (string) your application requires to be modified without the need of a new deployment.
+A _Setting_ is an essential bit of _ConfigCat_. It can be of multiple types, such as an on/off switch (bool), number (int, double) or any text (string) your application requires to be modified without the need for a new deployment.
```

Comment:
Maybe we could rephrase this as the current wording is a bit hard-to-follow:

```suggestion
A _Setting_ is an essential bit of _ConfigCat_. It can be of multiple types, such as an on/off switch (bool), number (int, double) or any text (string). You can use it to change your application's behavior remotely, without a new deployment.
```